src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,23 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object == this) {
+            return true;
+        }
+        if (object == null) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
```

Comment:
@SkaterWeather What is object will be Plane not Rectangle?

Reply:
I'am accidentally removed checking "inctanceof", sorry